ludwig/schema/config/utils.py

```diff
@@ -0,0 +1,74 @@
+import yaml
+
+from ludwig.constants import ACTIVE, COLUMN, INPUT_FEATURES, NAME, OUTPUT_FEATURES, PROC_COLUMN, TYPE
+from ludwig.features.feature_utils import compute_feature_hash
+from ludwig.schema.utils import convert_submodules
+from ludwig.utils.registry import Registry
+
+internal_output_config_registry = Registry()
+
+
```

Comment:
Added this registry for the output feature metadata classes because they each have a different validation metric. Can access them with this method.